pkgs/development/compilers/ghc/head.nix

```diff
@@ -5,15 +5,15 @@
   # If enabled GHC will be build with the GPL-free but slower integer-simple
   # library instead of the faster but GPLed integer-gmp library.
 , enableIntegerSimple ? false, gmp
+, version ? "8.3.20170808"
```

Comment:
Allowing to override the version number but not the actual version that’s used, i.e., `src` seems a bit weird.